DesignAutomationHandler2018/DesignAutomationHandler2018.csproj

```diff
@@ -45,6 +45,8 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Xml.Linq" />

```

Comment:
I think we want the same for 2019 csproj

Reply:
Agreed, I will update it when I am back.